tests/layer_tests/tensorflow_tests/test_tf_RandomUniform.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import tensorflow as tf
+
+from common.tf_layer_test_class import CommonTFLayerTest
+
+
+class TestTFRandomUniform(CommonTFLayerTest):
+    def create_tf_random_uniform_net(self, global_seed, op_seed, x_shape, min_val, max_val, input_type, ir_version):
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            tf_x_shape = x_shape.copy()
+            # reshaping
+            if len(tf_x_shape) >= 3:
+                tf_x_shape.append(tf_x_shape.pop(1))
+
+            x = tf.compat.v1.placeholder(input_type, x_shape, 'Input')
+            if global_seed is not None:
+                tf.random.set_seed(global_seed)
+            random_uniform = tf.random.uniform(x_shape, seed=op_seed, dtype=input_type, minval=min_val,
+                                               maxval=max_val) + x
+
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        # TODO: add reference IR net. Now it is omitted and tests only inference result that is more important
```

Comment:
I think that the reference IR is really simple so we can cover this:)

Reply:
Done.